### ISSUE TITLE
Include package import path into completion information

### ISSUE THREAD

**Issue by kostya-sh** [nsf/gocode#306]
It would be good to include package import path (e.g. math/big for Int) in completion information. This will allow editors to invoke `go doc` to display documentation for a selected completion candidate.

**Comment by nsf** [comment]
Good idea.

**Comment by bruno-medeiros** [comment]
Yup, this would be quite useful! I've been trying to get a way to get documentation visualization in goclipse, no luck so far (Go-oracle isn't much help here either).
